test/screenshot/lib/controller.js

```diff
@@ -275,12 +270,11 @@ class Controller {
    * Writes the given `testCases` to a `golden.json` file in `sourceDir_`.
```

Comment:
Update this comment if `sourceDir_` no longer exists.

Reply:
Done